docs/guides/jsdom.md

```diff
@@ -11,6 +11,31 @@ gets run *before* React's code is run.
 
 As a result, a standalone script like the one below is generally a good approach:
 
+`jsdom v10~`:
+
+```js
+/* setup.js */
+
+var { JSDOM } = require('jsdom');
+var jsdom = new JSDOM('');
```

Comment:
nits chage though. the latter parts is the same, so maybe better way to slim up?